tree/ntupleutil/v7/src/RNTupleImporter.cxx

```diff
@@ -350,12 +396,22 @@ ROOT::Experimental::RResult<void> ROOT::Experimental::RNTupleImporter::PrepareSc
 
 ROOT::Experimental::RResult<void> ROOT::Experimental::RNTupleImporter::Import()
 {
-   if (fDestFile->FindKey(fNTupleName.c_str()) != nullptr)
-      return R__FAIL("Key '" + fNTupleName + "' already exists in file " + fDestFileName);
+   for (const auto &sourceTree : fSourceTrees) {
+      Import(sourceTree);
```

Comment:
As long as we use the `RResult` signatures, we have to check here the return value and act upon errors. As discussed, in a follow-up PR we can remove `RResult` from the public interface and simply throw `RException`s.